src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEnginePublisher.java

```diff
@@ -207,6 +229,24 @@ public boolean isApplicable(Class<? extends AbstractProject> jobType) {
       return true;
     }
 
+    public static void setComputeClient(ComputeClient client) {
```

Comment:
This should be package scoped, also suggest adding \@VisibleForTesting annotation.

Reply:
Ah, thanks for the suggestion, I knew that public didn't feel right.

Reply:
Done